chipsec/hal/iobar.py

```diff
@@ -68,12 +68,23 @@ def get_IO_BAR_base_address( self, bar_name ):
         if 'register' in bar:
             bar_reg   = bar['register']
             if 'base_field' in bar:
-                base_field = bar['base_field']
-                base = self.cs.read_register_field( bar_reg, base_field, preserve_field_position=True )
-                empty_base = self.cs.get_register_field_mask( bar_reg, base_field, preserve_field_position=True )
+                try:
+                    base = self.cs.read_register_field( bar_reg, base_field, preserve_field_position=True )
```

Comment:
I don't see where base_field is defined. Did you mean bar['base_field']?

Reply:
Good catch, fixed